app/components/EmissionListItem/__tests__/__snapshots__/EmissionListItem.test.tsx.snap

```diff
@@ -76,9 +76,10 @@ exports[`EmissionListItem renders correctly if mitigated 1`] = `
         >
           <FormattedNumber
             maximumFractionDigits={2}
-            value={2.1}
+            value={4.6305000000000005}
```

Comment:
I would prefer to keep the kg values in snapshots across the entire project, or create two snapshots files like `EmissionListItem.metric.test.tsx.snap` and EmissionListItem.imperial.test.tsx.snap thanks to `EmissionListItem.metric.test.tsx` and `EmissionListItem.imperial.test.tsx`

Reply:
I put together a perhaps janky way to get this working, using the getUseMetricUnits selector to mock useSelector before each test in the affected snapshots. EmissionItemScreen I only mocked useSelector in this once per test since it uses other selectors, and this also required me to move the related line of code above all the other useSelector calls in order for it to receive the intended mocking